src/plugins/data/server/search/routes/search.ts

```diff
@@ -62,7 +62,7 @@ export function registerSearchRoute(
       try {
         const { withLongNumeralsSupport, ...response } = await selfStart.search.search(
           context,
-          { ...searchRequest, id },
+          { ...searchRequest, id, rawRequest: request },
```

Comment:
i wouldn't update this. when it's ppl, the whole request should be passed as long as you just send it

Reply:
You mean we check the `strategy` here, and when it's `ppl`, we send the whole request instead of request.body?

Reply:
The `client.asScoped(request)` we call in search strategy requires the http request object, currently I see no way to get it unless pass it specifically.